parsec/stubs/triopg/_triopg.pyi

```diff
@@ -18,6 +18,24 @@ class TrioConnectionProxy:
     async def __aenter__(self) -> TrioConnectionProxy: ...
     async def __aexit__(self, *exc: Any) -> Any: ...
 
+class TrioCursorProxy:
+    def __init__(self, asyncpg_cursor: Any) -> None: ...
+    async def fetch(self, *args: Any, **kwargs: Any) -> Any: ...
```

Comment:
Isn't it supposed to be an `Iterable`?

Reply:
I don't know

Reply:
Triopg, seems to internaly use this method which returns a list of records. https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg.pool.Pool.fetch